### ISSUE TITLE
High level segmentation model - Line parsing - mixed

### ISSUE THREAD

**Issue by HugoSchtr** [DataCatalogue/grobid-datacat#3]
With a more powerful PC, a __mixed__ model can be trained with line parsing, even using huge files. 

This model is mixed, all documents available were used for training. Document heterogeneity is then expected in the training set.

**Comment by HugoSchtr** [comment]
Augmenting training set worsened the model's performance: 

./gradlew train_datacat-segmenter

> Task :train_datacat-segmenter
sourceTEIPathLabel: /home/hscheith/dev/grobid/grobid-datacat/resources/dataset/datacat-segmenter/corpus/tei
sourceRawPathLabel: /home/hscheith/dev/grobid/grobid-datacat/resources/dataset/datacat-segmenter/corpus/raw
trainingOutputPath: /home/hscheith/dev/grobid/grobid-datacat/../grobid-home/tmp/datacat-segmenter17426930913151383059.train
evalOutputPath: null
529 tei files
14:28:53.331 [main] INFO org.grobid.trainer.AbstractTrainer - Processing: 12148-bpt6k9779630f.training.monograph.tei.xml
Total data found between CRF and TEI files 596 from total 814 examples.
[...]
14:29:37.012 [main] INFO org.grobid.trainer.AbstractTrainer - Processing: 12148-bpt6k9780628v.training.datacat.tei.xml
Total data found between CRF and TEI files 1893 from total 2138 examples.
        epsilon: 1.0E-7
        window: 50
        nb max iterations: 2000
        nb threads: 16
* Load patterns
* Load training data
* Initialize the model
* Summary
    nb train:    528
    nb labels:   24
    nb blocks:   1357685
    nb features: 32584992
* Train the model with l-bfgs
  [   1] obj=534757.66  act=3247713  err=18.36%/100.00% time=11.27s/11.27s
  [...]
  [1456] obj=2009.24    act=6858     err= 0.00%/ 2.08% time=8.87s/13965.29s
* Save the model
* Done
Model for datacat-segmenter created in 14018399 ms
sourceTEIPathLabel: /home/hscheith/dev/grobid/grobid-datacat/resources/dataset/datacat-segmenter/evaluation/tei
sourceRawPathLabel: /home/hscheith/dev/grobid/grobid-datacat/resources/dataset/datacat-segmenter/evaluation/raw
trainingOutputPath: /home/hscheith/dev/grobid/grobid-datacat/../grobid-home/tmp/datacat-segmenter1727358743364599982.test
evalOutputPath: null
117 tei files
18:22:31.883 [main] INFO org.grobid.trainer.AbstractTrainer - Processing: 12148-bpt6k9777334f.training.monograph.tei.xml
Total data found between CRF and TEI files 1249 from total 1651 examples.
18:22:31.960 [main] INFO org.grobid.trainer.AbstractTrainer - Processing: 12148-bpt6k9777569b.training.monograph.tei.xml
[...]
18:22:47.166 [main] DEBUG org.grobid.core.utilities.GrobidProperties - No configuration parameter defined for DeLFT engine for model datacat-segmenter
18:22:47.167 [main] INFO org.grobid.core.jni.WapitiModel - Loading model: /home/hscheith/dev/grobid/grobid-datacat/../grobid-home/models/datacat-segmenter/model.wapiti (size: 22215415)
[Wapiti] Loading model: "/home/hscheith/dev/grobid/grobid-datacat/../grobid-home/models/datacat-segmenter/model.wapiti"
Model path: /home/hscheith/dev/grobid/grobid-datacat/../grobid-home/models/datacat-segmenter/model.wapiti
Labeling took: 3947 ms

===== Field-level results =====

label                accuracy     precision    recall       f1           support

<advertisement>      99.41        0            0            0            4      
<annex>              97.54        46.43        56.52        50.98        23     
<back>               93.91        45.45        16.67        24.39        60     
<cover>              90.18        50           39           43.82        100    
<dedication>         99.8         66.67        66.67        66.67        3      
<index>              98.82        0            0            0            9      
<preface>            84.28        44.03        40.97        42.45        144    
<reference>          98.13        66.67        53.85        59.57        26     
<title>              87.33        43.93        40.52        42.15        116    
<toc>                97.05        47.62        34.48        40           29     
<unit>               83.6         34.81        37.3         36.02        126    

all (micro avg.)     93.64        43.5         37.66        40.37        640    
all (macro avg.)     93.64        40.51        35.09        36.91        640    

===== Instance-level results =====

Total expected instances:   117
Correct instances:          11
Instance-level recall:      9.4

**Comment by HugoSchtr** [comment]
New labels for training high segmentation label are now used with the new debugged high level segmentation module.
